Makefile

```diff
@@ -84,5 +84,10 @@ install-controller-gen:
 # generate crds with controller-gen
 generate-operator-crds: install-controller-gen 
 	$(CONTROLLER_GEN_BIN) $(CRD_OPTIONS) paths=api/v1/deployment_types.go paths=api/v1/groupversion_info.go output:crd:dir=$(CSI_CHART_CRDS_PATH)
-lint:
```

Comment:
Do we call it in operator build automation? Maybe stay `lint`=`lint-code`+`lint-charts` target?